source/patterns/@aws-solutions-constructs/aws-lambda-kinesisfirehose/test/lambda/index.mjs

```diff
@@ -0,0 +1,32 @@
+import { FirehoseClient, PutRecordCommand } from "@aws-sdk/client-firehose";
+const client = new FirehoseClient({});
+
+export const handler = async(event) => {
```

Comment:
Nice test that auto puts records on the stream! Makes the validation a whole lot easier having the test lambda function do it for us!